### ISSUE TITLE
The TinyMCE accessibility checker doesn't validate the color contrast with Tiny Font Color

### ISSUE THREAD

**Issue by decristm** [bfh/moodle-tiny_fontcolor#9]
Hi!
In the documentation of TinyMCE here : https://docs.moodle.org/402/en/TinyMCE_editor#Accessibility_checker, it says it checks the contrast of font color and background color with the WCAG. 
But when I test it with Tiny Font Color, it doesn't work.
Example :
![image](https://github.com/bfh/moodle-tiny_fontcolor/assets/74793913/6eaaf1a4-5947-4c92-8f97-7ee3c2f885e8)
![image](https://github.com/bfh/moodle-tiny_fontcolor/assets/74793913/e50ea0c4-409e-4bda-aaad-5f850715a7c4)

It would be better for accessibilty that it works together. Thanks in advance!

**Comment by srobotta** [comment]
Hi,

can you please provide the exact color codes (or maybe the exact styles that are attached with this tag)? I am afrait that is not an issue with the colorcode plugin.

![2023-07-04_16-46](https://github.com/bfh/moodle-tiny_fontcolor/assets/5969124/5f6bc459-cec5-4a33-afdf-e18b2ffc6a37)

In this screenshot the "Wege" and "chemins" are reported by the accessibility tool while the next contrast in "mènent" is not reported. So the tool works in general I would say.

Thank you.

**Comment by srobotta** [comment]
There hasn't been any feedback for the last 3 months, so I will close this as "not a bug" because I can't reproduce it.
